### ISSUE TITLE
Add a page to test the QT web browser

### ISSUE THREAD

**Issue by pontusr** [victronenergy/gui-v2#822]
Hi!

I understand there may be performance issues for running the QT web browser component on Cerbo hardware. Have this already been tested and ruled out as a possibility? If not, it seems relatively trivial to add a new browser page (disabled by default, enabled through settings). This would allow the community to test this and provide feedback for different scenarios - for my part I would like to test some very simple pages generated from Node-Red Dashboard 2.0 which works great in the VRM.

The initial url to load could be defined in the settings. Another option could be to mimic the "Venus OS Large menu" from the https://vrm.victronenergy.com/installation/XXXXXX/proxy page to provide a menu of detected local pages on Node-Red activated setups.

**Comment by chriadam** [comment]
Would require shipping QtWebEngine with the device image.  Unsure of need / priority on this one, marking as low prio for now, and pushing back to v1.3.0.

**Comment by chriadam** [comment]
I don't think we intend to ship QtWebEngine in gui-v2, so closing as wontfix.
